### PR TITLE
EC-40: Create abstraction for CLI to remove unnecessary liability from modules (CLI name)

### PR DESCRIPTION
**Pull request by dmytrostriletskyi**
I have created [setup cli name](https://github.com/essentiaone/ess-base/pull/37/files#diff-d43e4925233646b339cf5faff09dadd8R124) functionality to specify CLI name of module (class) as part of another abstraction (decorator) instead of explicit setting class variable within class.

We migrate from specifying zone of responsibility within class:
```python
class Bitcoin:

    cli_name = 'bitcoin-wallet'
    ...
```

To do it under class:
```python
@setup_cli_name('bitcoin-wallet')
class Bitcoin:
    ...
```

### CODE REVIEW

File: ess_base/modules/payment/omisego/omisego.py [hunk 1]
```diff
@@ -8,14 +8,15 @@
 import modules.payment.omisego.services.recipient
 import modules.payment.omisego.services.transfer
```
Comment (a-ruban) [inline ess_base/modules/payment/omisego/omisego.py]: @dmytrostriletskyi please, delete unappropriate blank line.

File: ess_base/modules/wallet/bitcoin/bitcoin.py [hunk 2]
```diff
@@ -5,15 +5,15 @@
 import modules.wallet.bitcoin.services.transaction
```
Comment (a-ruban) [inline ess_base/modules/wallet/bitcoin/bitcoin.py]: @dmytrostriletskyi please, remove blank line.

File: ess_base/modules/payment/omisego/omisego.py (hunk 1 above)
Comment (dmytrostriletskyi) [reply ess_base/modules/payment/omisego/omisego.py]: @a-ruban, done.

File: ess_base/modules/wallet/bitcoin/bitcoin.py (hunk 2 above)
Comment (dmytrostriletskyi) [reply ess_base/modules/wallet/bitcoin/bitcoin.py]: @a-ruban, done. Also want to clear the import of services. Services are special abstraction. We separate it by blank lines at the end of the local import. Thanks. :)

**Pull request merged**